compiler/src/iree/compiler/Codegen/Common/test/math_transform.mlir

```diff
@@ -66,22 +66,20 @@ func.func @erf_fastmath(%arg0: f32) -> f32 attributes {
 
   // CHECK-DAG: arith.constant 1.000000e+00 : f32
 
-  // Then region coefficients (|x| < 1.0)
-  // CHECK-DAG: arith.constant -5.61801775E-4 : f32
-  // CHECK-DAG: arith.constant 0.00491381623 : f32
-  // CHECK-DAG: arith.constant -0.0267075151 : f32
-  // CHECK-DAG: arith.constant 0.112800106 : f32
-  // CHECK-DAG: arith.constant -0.376122952 : f32
-  // CHECK-DAG: arith.constant 0.128379107 : f32
-
-  // Else region coefficients (|x| >= 1.0)
-  // CHECK-DAG: arith.constant 1.69988107E-5 : f32
-  // CHECK-DAG: arith.constant -3.78677854E-4 : f32
-  // CHECK-DAG: arith.constant 0.00385781587 : f32
-  // CHECK-DAG: arith.constant -0.0241816975 : f32
-  // CHECK-DAG: arith.constant 0.106668264 : f32
-  // CHECK-DAG: arith.constant 0.634933292 : f32
+  // All coefficients in order as they're consumed.
   // CHECK-DAG: arith.constant 0.128689408 : f32
+  // CHECK-DAG: arith.constant 0.634933292 : f32
+  // CHECK-DAG: arith.constant 0.106668264 : f32
+  // CHECK-DAG: arith.constant -0.0241816975 : f32
+  // CHECK-DAG: arith.constant 0.00385781587 : f32
+  // CHECK-DAG: arith.constant -3.78677854E-4 : f32
+  // CHECK-DAG: arith.constant 1.69988107E-5 : f32
+  // CHECK-DAG: arith.constant 0.128379107 : f32
+  // CHECK-DAG: arith.constant -0.376122952 : f32
+  // CHECK-DAG: arith.constant 0.112800106 : f32
+  // CHECK-DAG: arith.constant -0.0267075151 : f32
+  // CHECK-DAG: arith.constant 0.00491381623 : f32
+  // CHECK-DAG: arith.constant -5.61801775E-4 : f32
```

Comment:
Why do we keep them in order as they're consumed? The previous form looks better to me, because you can map them to the code eaiser.